propagators/baggage.go

```diff
@@ -26,7 +26,7 @@ import (
 
 // Temporary header name until W3C finalizes format.
```

Comment:
this comment is now out of date yeah?

Reply:
Thanks for catching, I've removed the comment as well.